src/protocol/ConnectionType.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * RakLib network library
+ *
+ *
+ * This project is not affiliated with Jenkins Software LLC nor RakNet.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ */
+
+declare(strict_types=1);
+
+namespace raklib\protocol;
+
+#include <rules/RakLibPacket.h>
+
+use raklib\RakLib;
+
+class ConnectionType{
+	
+	public const MAX_METADATA_VALUES = 0xff;
+	
+	public static function createMetaData(string... $metadata) : array{
```

Comment:
Inconsistent code style with other parts of @pmmp

Reply:
I'm not sure how this is inconsistent with other parts of PMMP, could you elaborate please?

Reply:
In PHP, the `...` is defined to be on the left of the parameter name, not to be on the right of the data type. We usually put the space between the type and the `...`.

Reply:
Alright, thanks for the info!

Reply:
I added the space like you suggested! :)